src/app/routes/index.js

```diff
@@ -1,6 +1,7 @@
 import Article from '../containers/Article';
 import FrontPage from '../containers/FrontPage';
 import MediaPage from '../containers/MediaPage';
```

Comment:
Do we still need this? Aren't we importing the `MediaPage` in the new `CpsAssetPage` container?

Reply:
We don't but this was meant to be a quick POC. Will tidy things up and put it up for a review. Thanks for taking a look.